package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "TheForemanDevDeps",
-  "version": "1.18.0",
+  "version": "1.19.0-develop",
```

Comment:
no idea if `-develop` hurts there in any way...

Reply:
I looked, it was 1.17.0 before, so let's play it safe - changed to 1.19.0